### PR TITLE
Router: implement alignment of closing paren

### PR DESCRIPTION
**Pull request by kitbellew**
Helps with #2562.

### CODE REVIEW

File: scalafmt-tests/src/test/resources/scala3/TypeLambda.stat [hunk 1]
```diff
@@ -12,6 +12,29 @@ maxColumn = 30
...
+      TheOnlyTypeInTheTuple,
+      TheOnlyTypeInTheTuple
+  )
+<<< type lambda long, align open
+maxColumn = 30
+align.openParenCallSite = true
+===
+type   Tuple   =  [  TheOnlyTypeInTheTuple ] =>> (TheOnlyTypeInTheTuple, TheOnlyTypeInTheTuple)
+>>>
+type Tuple =
```
Comment (tgodzik) [inline scalafmt-tests/src/test/resources/scala3/TypeLambda.stat:26]: The result here seems to be identical to the next test and the previous one. What is this actually testing?

Shouldn't the last one be:
```scala
type Tuple = 
  [TheOnlyTypeInTheTuple] = (
                            TheOnlyTypeInTheTuple,      
                            TheOnlyTypeInTheTuple  
                            )
```

Comment (kitbellew) [reply scalafmt-tests/src/test/resources/scala3/TypeLambda.stat:26]: yep... let me check, perhaps maxColumn should be larger.

Comment (kitbellew) [reply scalafmt-tests/src/test/resources/scala3/TypeLambda.stat:26]: found it. it was a test for the next batch of commits which i haven't submitted yet :) will move it there.

Comment (kitbellew) [reply scalafmt-tests/src/test/resources/scala3/TypeLambda.stat]: are we good to go?

**Review by tgodzik — APPROVED** [review]
LGTM! Sorry, I forgot about this one.

**Pull request merged**